### PR TITLE
Adding CMakeLists.txt. Updating model.cc for relative paths

### PR DESCRIPTION
**Pull request by PinkySan**
I added a CMakeLists.txt for using your example with CMake (i am not that familiar with bazel and my companie's workflow prescribes CMake).

For the CMake-Example you need the actual master branch or at least a version, which contains [PR#14554](https://github.com/tensorflow/tensorflow/pull/14554).

**Remarks:** This is only tested with windows

### CODE REVIEW

File: model.cc [hunk 1]
```diff
@@ -11,7 +11,7 @@ using namespace std;
 
 int main() {
 
-  DataSet data_set("/Users/fco/Documents/personnel/projects/tensorflow/tensorflow/cc/models/normalized_car_features.csv");
+  DataSet data_set("normalized_car_features.csv");
```
Comment (PinkySan) [inline model.cc:14]: I guess this will only be valid in my case. I copy the file into my binary folder

**Comment by theflofly** [comment]
Thanks for the PR, I will check in more details next week and merge.

File: model.cc (hunk 1 above)
Comment (theflofly) [reply model.cc:14]: Yeah, I am wondering how to do it using bazel.

Comment (theflofly) [reply model.cc:14]: FYI I am still searching a way to copy the file in the binary dir.

Comment (theflofly) [reply model.cc:14]: Finally found a way to do it with bazel, should be merged soon.

**Comment by theflofly** [comment]
Merged, I tried to build on mac osx but I add several errors, it seems to work well only on Windows. I assume that it is working. Thanks for the pull request.

**Pull request merged**